x-pack/plugins/enterprise_search/public/applications/workplace_search/components/shared/source_row/source_row.tsx

```diff
@@ -145,23 +149,23 @@ export const SourceRow: React.FC<SourceRowProps> = ({
             onChange={(e: EuiSwitchEvent) => onSearchableToggle(id, e.target.checked)}
             disabled={!supportedByLicense}
             compressed
-            label="Source Searchable Toggle"
+            label={SOURCE_ROW_SEARCHABLE_TOGGLE_LABEL}
             showLabel={false}
             data-test-subj="SourceSearchableToggle"
           />
         </EuiTableRowCell>
       )}
       <EuiTableRowCell align="right">
         <EuiFlexGroup justifyContent="flexEnd" alignItems="center" gutterSize="s">
-          {showFix && <EuiFlexItem grow={false}>{fixLink}</EuiFlexItem>}
+          {showReauthenticate && <EuiFlexItem grow={false}>{fixLink}</EuiFlexItem>}
```

Comment:
nit: Could you rename `fixLink` variable as well?

Reply:
Yep! https://github.com/elastic/kibana/pull/103547/commits/2f6badfbaa3020d4611d85cc03daf82951cafe9a